### PR TITLE
DBZ-4260 Blog post for Debezium UI topic auto-creation

### PR DESCRIPTION
**Pull request by mdrillin**
blog post for topic group support in the UI

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@rk3rn3r, @mdrillin, it's getting late, so let's try and wrap this one up for tomorrow then. Thx!

**Comment by gunnarmorling** [comment]
@mdrillin, hey, what's needed to undraft this one? Does it have the right video URL already? If not, @rk3rn3r, can you help with that? Let's aim for Wednesday with this, i.e. can you update the date and file name accordingly, @mdrillin?

File: _posts/2021-11-29-debezium-ui-topic-groups.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  Debezium UI support for Topic Group automatic creation
+date:   2021-11-29
+tags: [ releases, mysql, postgres, mongo, debezium-ui ]
+author: mdrilling
+---
+
+The Debezium UI team is pleased to announce support for automatic creation of Topic Groups in the Debezium UI!  Our goal with the Debezium graphical user interface is to allow users to set up and operate connectors more easily.  To that end, we have added support for Topic Groups to the UI.  Read futher for more information, and for a video demo of the new feature!
```
Comment (gunnarmorling) [inline _posts/2021-11-29-debezium-ui-topic-groups.adoc]: That's rather repetitive with the previous post. Can you reword a bit, e.g. "Continuing our efforts to help users with setting up...".

File: _posts/2021-11-29-debezium-ui-topic-groups.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
++++<!-- more -->+++
+
+Let's take a closer look at Debezium UI support for the automatic topic creation feature:
+
+== Automatic Topic Creation
+
+Beginning with Kafka 2.6.0, Kafka Connect comes with built-in topic creation for connector topics. If automatic topic creation is enabled, the Debezium UI now allows you to configure connector topics using the UI.
```
Comment (gunnarmorling) [inline _posts/2021-11-29-debezium-ui-topic-groups.adoc]: Here we should have a few sentences motivating topic creation settings, for those who don't know what they are good for. @rk3rn3r's [post](https://debezium.io/blog/2020/09/15/debezium-auto-create-topics/) can serve as inspiration.

File: _posts/2021-11-29-debezium-ui-topic-groups.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+* Incorporation of more Debezium connector types, such as the ones for SQL Server and Oracle
+* Addition and improvement of connector metrics and monitoring
+* Add capability for viewing and editing connector properties after creation
+* ...And more!
+
+We'd also be very happy to learn about your requirements and feedback on the Debezium UI.
+Please let us know in the comments below, or send a message to our https://groups.google.com/g/debezium[mailing list].
+
+_A big thank you to the team who have contributed in many ways: Ashique Ansari, Indra Shukla, René Kerner and Gunnar Morling!_
```
Comment (gunnarmorling) [inline _posts/2021-11-29-debezium-ui-topic-groups.adoc]: That's all very similar to the earlier post? Let's better avoid the duplication and either rephrase or link to the earlier post, e.g. in terms of the next features planned to be added.

**Review by gunnarmorling — COMMENTED** [review]
LGTM overall, @mdrillin. A few comments inline.

**Comment by mdrillin** [comment]
> @mdrillin, hey, what's needed to undraft this one? Does it have the right video URL already? If not, @rk3rn3r, can you help with that? Let's aim for Wednesday with this, i.e. can you update the date and file name accordingly, @mdrillin?

@gunnarmorling sure, I can update date and file for Wednesday and make some of the suggested text changes.  The topic group video will need to be uploaded to youtube channel and then the link updated in the article

**Comment by rk3rn3r** [comment]
@mdrillin All videos are already on the YT channel. I shared the links on chat, back when I created the channel. You can use the link from chat. I will set the video to public when we merge the PR and publish the blog post.

**Comment by mdrillin** [comment]
@gunnarmorling @rk3rn3r 
- renamed file and dated 2021-12-01
- Added more description content (borrowed most of the wording from documentation, etc).
- Removed some content that was repeated from other article
- updated the youtube link (sorry, I forgot the video was there but just hidden)

**Comment by gunnarmorling** [comment]
Applied; thanks a lot, @mdrillin!

**Pull request merged**